kubernetes-client/src/test/java/io/fabric8/kubernetes/client/impl/DefaultKubernetesClientTest.java

```diff
@@ -13,11 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.fabric8.kubernetes.client;
+package io.fabric8.kubernetes.client.impl;
```

Comment:
The test class name should probably be renamed

Reply:
Done